README.md

```diff
@@ -74,19 +74,12 @@ Let's say a user wants to export 4 fields: `name`, `cost`, `translations_spanish
 {
     "test": {
         "products": {
+            "destination_name": "my_products",
             "fields": {
-                "name": {
-                    "source": "name"
-                },
-                "cost": {
-                    "source": "cost"
-                },
-                "translations_spanish": {
-                    "source": "translations.spanish"
-                },
-                "translations_french": {
-                    "source": "translations.french"
-                }
+                "name": "my_name",
```

Comment:
does this need to be updated? i think `name` should point to a dict, which contains a KV pair `destination_name` -> `my_name`